Stack/Opc.Ua.Core/Stack/Tcp/UaSCBinaryClientChannel.cs

```diff
@@ -224,7 +224,7 @@ public async Task CloseAsync(int timeout, CancellationToken ct = default)
             {
                 try
                 {
-                    await operation.EndAsync(timeout, true, ct).ConfigureAwait(false);
+                    _ = await operation.EndAsync(timeout, false, ct).ConfigureAwait(false);
```

Comment:
Not throwing on error (true -> false) should probably invalidate the whole try/catch, could it all be removed? Seems like it is just used for logging anyway, which probably already happend.

Reply:
it can still be cancelled